api/controllers/StatsController.js

```diff
@@ -0,0 +1,149 @@
+const dayjs = require('dayjs');
+const isBetween = require('dayjs/plugin/isBetween');
+dayjs.extend(isBetween);
+
+const Result = {
+  WON: 1,
+  LOST: 2,
+  INCOMPLETE: -1,
+};
```

Comment:
Used in front and backend so that would be helpful. Might run into a really awkward issue that the frontend uses esmodules and the backend uses commonjs/require syntax...

Reply:
done